Documents/API.md

```diff
@@ -132,30 +158,50 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
     * `POST` - Upload HMDA data to submission
 
 
-* `/institutions/<institution>/summary`
-    * `GET` - Summary for Financial Institution, including filing information
+* `/institutions/<institution>/filings/<period>/submissions/<submissionId>/edits`
+    * `GET`  - List of all edits for a given submission, grouped by edit type
 
     Example response, with HTTP code 200:
 
     ```json
     {
-      "id": "12345",
-      "name": "First Bank",
-      "filings": [
-        {
-          "period": "2017",
-          "institutionId": "12345",
-          "status": "not-started"
-        },
-        {
-          "period": "2016",
-          "institutionId": "12345",
-          "status": "completed"
-        }
-      ]
+      "syntactical": {
+        "edits": [
+          {
+            "edit": "S025",
+            "lars": [
+              {
+                "lar": {"loanId": "s1"}
```

Comment:
(Adding more specific details to my previous comment) 

The API response in this PR has 
``` json
"loanId": "s1"
```
rather than 
``` json
"lar": {"loanId": "s1"}
```

Which should it be?